launch/launch/utilities/__init__.py

```diff
@@ -27,7 +26,6 @@
     'is_a',
     'is_a_subclass',
     'isclassinstance',
-    'create_future',
```

Comment:
I should point out that the removal of this implementation is technically removing something from the public launch API, and thus in theory we should deprecate it first.  However, I looked through all of the released packages on Rolling, and the only one that used this was `launch_ros`.  Therefore, I think it is safe to say that this is mostly an implementation detail, and we can safely remove it.